src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitationsForExport/GetInvitationsForExcelQueryHandler.cs

```diff
@@ -122,15 +121,36 @@ select new
 
             foreach (var participant in participants)
```

Comment:
remove whole foreach-block, rewrite method to just return the statement at line 109 as body. expr:
![image](https://user-images.githubusercontent.com/19545528/232982777-d758e1a7-62e6-4969-99cd-4c0036a3761d.png)